pkg/odo/cli/component/create.go

```diff
@@ -324,6 +333,7 @@ func NewCmdCreate() *cobra.Command {
 	componentCreateCmd.Flags().StringVar(&cpuMax, "max-cpu", "", "Limit maximum amount of cpu to be allocated to the component. ex. 1")
 	componentCreateCmd.Flags().StringSliceVarP(&componentPorts, "port", "p", []string{}, "Ports to be used when the component is created (ex. 8080,8100/tcp,9100/udp")
 	componentCreateCmd.Flags().StringSliceVar(&componentEnvVars, "env", []string{}, "Environmental variables for the component. For example --env VariableName=Value")
+	componentCreateCmd.Flags().BoolVarP(&componentWait, "wait", "w", false, "If present, wait until the component is ready")
```

Comment:
```If present``` in the message is unnecessary overloading the same sense IMO. It is understandable how flags work.

Reply:
I agree, just do: `Wait until the component is ready`